registry/storage/walk.go

```diff
@@ -26,7 +27,12 @@ func Walk(ctx context.Context, driver storageDriver.StorageDriver, from string,
 	if err != nil {
 		return err
 	}
+	sort.Stable(sort.StringSlice(children))
```

Comment:
Cost is here: https://godoc.org/sort#Stable. Click through to code to see the full cost analysis. I don't think we'll hit asymptotic bounds very often.